lib/compress/index.js

```diff
@@ -4380,15 +4380,11 @@ function inline_object_prop_spread(props, compressor) {
                 props.splice(i, 1, ...expr.properties);
                 // Step back one, as the property at i is now new.
                 i--;
-            } else if (expr instanceof AST_Constant
+            } else if (expr.is_constant()
```

Comment:
You could also special case a unary prefix of a string.

Reply:
Sorry? `.is_constant()` already returns `true` for unary prefixed strings. It's covered in the test. Did I get you wrong?

Reply:
Ha, I missed that in the test case.